src/github.com/matrix-org/dendron/dendron/main.go

```diff
@@ -41,7 +43,7 @@ var (
 	listenKeyFile          = flag.String("key-file", "", "TLS Private Key. The private key for the certificate. This must be set if listening for HTTPS requests")
 	pusherConfig           = flag.String("pusher-config", "", "Pusher worker config")
 	synchrotronConfig      = flag.String("synchrotron-config", "", "Synchrotron worker config")
-	synchrotronURLStr      = flag.String("synchrotron-url", "", "The HTTP URL that the synchrotron will listen on")
+	synchrotronURLStr      = flag.String("synchrotron-url", "", "Comma separated list of HTTP URLs that the synchrotron will listen on")
```

Comment:
Would it not be "better" to allow you to specify this flag multiple times? Or does the library not really support it?